service/start-acs.sh

```diff
@@ -32,14 +33,18 @@ main() {
                 JAVA_DEBUG_OPTS='-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005'
                 shift
                 ;;
+            'disable-predix-log-layout')
+                LOGGING_OPTS="-Dlog4j.debug -Dlog4j.configuration=file:${DIR}/src/main/resources/log4j-dev.xml"
+                shift
```

Comment:
This flag swaps `PredixLayout` for the original log4j layout to provide more human-readable logs in the locally-spun-up ACS service (useful for debugging). To make this more clear, I've renamed it to `human-readable-logging`.